src/webviews/react/diff-viewer/diff-viewer.css

```
@@ -408,14 +408,39 @@
+
+   Scoped to the ONE-SIDED states on purpose. A row whose counterpart is padding also exists
+   inside a two-sided `.diff-segment-modified` hunk with unequal line counts, and there the
+   full-length mark is the only thing separating "this line was removed" from "this line was
+   changed" -- the block wash there is --diff-info for both panes. That is also why this is not
+   fixed in `WordDiffLine` by skipping an empty counterpart: the component can see that a row
+   has nothing opposite it, but not which block it is sitting in, and the mark is meaningless in
+   one and load-bearing in the other. */
 .diff-viewer .diff-segment-inserted {
     --diff-segment-hue: var(--diff-ok);
+    --diff-word-wash: transparent;
     box-shadow: var(--diff-segment-shadow);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)